scripts/user_data_base.ps1

```diff
@@ -0,0 +1,163 @@
+<powershell>
+#User Data script
+$ErrorActionPreference = "Stop";
+
+Import-Module "C:\Program Files (x86)\AWS Tools\PowerShell\AWSPowerShell\AWSPowerShell.psd1"
```

Comment:
I could be wrong but I don't think this is actually needed anymore, I think it's automatically loaded now.